Sources/MapboxCoreNavigation/ActiveNavigationEventDetails.swift

```diff
@@ -65,6 +68,8 @@ struct ActiveNavigationEventDetails: NavigationEventDetails {
         originalRequestIdentifier = session.originalRoute?.routeIdentifier
         requestIdentifier = dataSource.routeProgress.route.routeIdentifier
                 
+        appMetadata = userInfo
```

Comment:
Maybe It's better to use `appMetadata` for the argument name in the constructor. Or use `useInfo` for the property name.

Reply:
There was some back and forth about this naming. We decided to use userInfo for the user-facing variable name, but the event naming needs to match the event schema which uses appMetadata.

Reply:
Here I'm talking about consistency between initializer arguments and properties. At the moment this (and similar) struct is not a part of the public API, so both namings are OK, but it would be better to make it sonsistent.